src/extension/runner.ts

```diff
@@ -440,6 +451,13 @@ export class GrpcRunnerProgramSession implements IRunnerProgramSession {
   }
 
   open(): void {
+    // Workaround to force terminal to close if opened after early exit
+    // TODO(mxs): find a better solution here
```

Comment:
Should we port this into an issue before we land the PR?

Reply:
I'll add an issue for it.

I don't think it's very high priority, basically just the `_onDidClose` gets fired twice for an interactive task. I think most things that listen to that are idempotent so it should be fine.

Reply:
#286